homeassistant/components/islamic_prayer_times/config_flow.py

```diff
@@ -0,0 +1,65 @@
+"""Config flow for Islamic Prayer Times integration."""
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import callback
+
+from .const import CALC_METHODS, CONF_CALC_METHOD, DEFAULT_CALC_METHOD, DOMAIN, NAME
```

Comment:
You need to add `# pylint: disable=unused-import` to solve the pylint error. See other config flows for reference.